src/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
```

Comment:
> - [ ] Please I suggest you use require_relative instead of require as the file path throws an error when checking the result.👍
> 
> ![tu2](https://github.com/fatima-najafi/oop-school-library/assets/40009354/1f263deb-293e-4750-8780-c5fdc91b5c92)
> 
> ```suggestion
> require_relative 'decorator'
> ```